addons/mail/static/src/activity/activity_menu.scss

```diff
@@ -11,3 +13,12 @@
         background-color: map-get($theme-colors, "light");
     }
 }
+
+.o-mail-no-activity {
+    cursor: initial;
+    align-items: center;
+    color: grey;
```

Comment:
Prefer a `text-muted` classname, which is more the intention and is dark-theme compliant when enterprise branch will work eventually.